platform/public_api/users/index.json

```diff
@@ -0,0 +1,402 @@
+        }
+      ]
+    }
+  },
+  "responses": {
+    "Bad_Request": {
+      "description": "Bad Request"
+    },
+    "Not_Found": {
+      "description": "Not Found"
+    }
+  },
+  "paths": {
+    "/api/users/v2": {
+      "post": {
+        "tags": [
+          "Users v2"
+        ],
+        "summary": "Create user",
+        "operationId": "api_user_add_v2",
+        "consumes": [
+          "application/vnd.api+json"
+        ],
+        "produces": [
+          "application/vnd.api+json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "user data",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/JsonApi_User_Create"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/JsonApi_User_Object"
+            }
+          },
+          "400": {
+            "$ref": "#/responses/Bad_Request"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Users v2"
+        ],
+        "summary": "Get users list",
+        "operationId": "api_users_list_v2",
+        "produces": [
+          "application/vnd.api+json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/definitions/JsonApi_User"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/api/users/v2/{uuid}": {
+      "parameters": [
+        {
+          "name": "uuid",
+          "in": "path",
+          "type": "string",
+          "format": "uuid",
+          "required": true
+        }
+      ],
+      "get": {
+        "tags": [
+          "Users v2"
+        ],
+        "summary": "Get details of an user",
+        "operationId": "api_users_user_details_v2",
+        "produces": [
+          "application/vnd.api+json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Details of an user",
+            "schema": {
+              "$ref": "#/definitions/JsonApi_User_Object"
+            }
+          }
+        }
+      },
+      "delete": {
+        "summary": "Delete user",
+        "operationId": "api_user_delete_v2",
+        "tags": [
+          "Users v2"
+        ],
+        "produces": [
+          "application/vnd.api+json"
+        ],
+        "responses": {
+          "204": {
+            "description": "Successful deletion of an user"
+          },
+          "400": {
+            "$ref": "#/responses/Bad_Request"
+          },
+          "404": {
+            "$ref": "#/responses/Not_Found"
+          }
+        }
+      }
+    },
+    "/api/users/v2/{uuid}/change-role": {
```

Comment:
Not a great description of parameter. It tells me format of parameter, but not what it is used for. I also see that it's definition doesn't have any description, so only way to identify what it does is context.